### ISSUE TITLE
 "Cannot read properties of undefined (reading 'startTime')"" error on testing a website

### ISSUE THREAD

**Issue by sreedhanyagn** [sitespeedio/browsertime#1913]
### Your question

I am testing sitespeed.io on a linux instance and I am using sitespeedio docker for the entire setup. 
The command I ran is :
 docker run --rm -v "$(pwd):/sitespeed.io" sitespeedio/sitespeed.io:26.1.0  <urls.txt> --slug chrome_browser_test --graphite.addSlugToKey true -n 1 --graphite.host=<private_ip> --grphite.port 80 -b chrome

I have provided multiple urls in the url text file and it is giving the following error for some of the urls after the command is executed.

 "performance": {
    "firstContentfulPaint": "Cannot read properties of undefined (reading 'startTime')",
    "longTasks": "Cannot read properties of undefined (reading 'startTime')"
  }

Is there anything we need to configure to set the startTime?

**Comment by soulgalore** [comment]
Hi @sreedhanyagn thanks for creating the issue! Hmm I think that data comes from the Coach and it means that you don't have a First Contentful Paint for that page. Let me add better error handling for that in the Coach, but then it's also strange why Chrome doesn't create that metric for your page?

**Comment by sreedhanyagn** [comment]
Thanks @soulgalore  for the prompt response. 
I have also scheduled a Jenkins job to run the docker command in every 3 hrs and I have observed that the error is not produced in some of the runs. If it was a problem with page, it should be a consistent right? 

The metrics that are not displayed are FirstVisualChange and LastVisualChange, attaching the screenshot below,
![image](https://user-images.githubusercontent.com/48643653/225461377-e9bd6903-2aae-46c0-b019-1f2501b58396.png)
Not sure if it is really an issue, because the value populated for some runs which gave the above error.
